### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mirons-google**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,139 +1,79 @@
...
-    We strive to have high and useful coverage by unit tests.  If your change
-    involves substantial logic, we will request that you write applicable unit
-    tests.
+         # Clone your forked repository into local directory.
+         git clone FORKED_REPOSITORY_URL
 
-    Our unit tests are written with google-apputils basetest framework.
-    See a [basic example] of how to use it, in the "Google-Style Tests" section.
+         # Navigate to your newly cloned directory.
+         cd <path to you local cloned repository>
```
Comment (carise) [inline CONTRIBUTING.md]: Lines 36 and 37 should sound more consistent -- directory or repository?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,139 +1,79 @@
...
+  1. Create a new local development branch for your feature or bug fix:
 
-9. Merging your PR.
+         git checkout -b YOUR_DEVELOPMENT_BRANCH
 
-    Once your PR is approved and all merge conflicts are resolved, we will merge your PR.
+  1. Create your change.
+    - Don't incorporate multiple changes in one PR. A change should be a logical, self-contained unit of work, feature, or fix. This simplifies troubleshooting and rollbacks.
+    - Learn how to execute [Inventory](inventory-quickstart), [Scanner](scanner-quickstart), or [Enforcer](enforcer-quickstart).
+  1. Create your test.
```
Comment (carise) [inline CONTRIBUTING.md]: "Create tests for your change."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,139 +1,79 @@
...
+    - Unit tests should provide high and useful coverage. If your change involves substantial logic, we'll ask you to write applicable unit tests.
```
Comment (carise) [inline CONTRIBUTING.md]: 2nd sentence: "You should write applicable unit tests for your changes, especially for changes involving substantial logic."

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,139 +1,79 @@
...
+    - Unit tests should be written with google-apputils basetest framework. For an example of how to use the framework, see [Google-Style Tests](https://pypi.python.org/pypi/google-apputils).
```
Comment (carise) [inline CONTRIBUTING.md]: Please remove this line; we no longer use apputils basetest.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,139 +1,79 @@
...
-[Google Python Style Guide]: https://google.github.io/styleguide/pyguide.html
-[Inventory]: https://googlecloudplatform.github.io/forseti-security/modules/core/inventory/
-[Scanner]: https://googlecloudplatform.github.io/forseti-security/modules/core/scanner/
-[Enforcer]: https://googlecloudplatform.github.io/forseti-security/modules/core/enforcer/
-[basic example]: https://pypi.python.org/pypi/google-apputils
-[Instructions to run the tests.]: https://googlecloudplatform.github.io/forseti-security/testing/
-[forseti-security@google.com]: mailto:forseti-security@google.com
+  1. Open a [pull request](https://help.github.com/articles/about-pull-requests/) to begin the code review.
+    - All submissions, including submissions by project members, require a code review.
+    - The GitHub UI will display any merge conflicts you need to resolve.
```
Comment (carise) [inline CONTRIBUTING.md]: The Github UI will display whether you have any merge conflicts that need to be resolved.

**Comment by mirons-google** [comment]
Thanks for the review! I implemented the requested changes.

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**